app/models/manageiq/providers/amazon/cloud_manager/event_catcher/stream.rb

```diff
@@ -159,12 +160,29 @@ def queue_url_to_arn(queue_url)
   # @param [Aws::SQS::Types::Message] message
   def parse_event(message)
     event = JSON.parse(JSON.parse(message.body)['Message'])
-    $log.info("#{log_header} Found SNS Message with message type #{event["messageType"]}")
-    return unless event["messageType"] == "ConfigurationItemChangeNotification"
+
+    if event["messageType"] == "ConfigurationItemChangeNotification"
+      # Aws Config Events
+      event["eventType"]    = parse_event_type(event)
+      event["event_source"] = :config
+    elsif event.fetch_path("detail", "eventType") == "AwsApiCall"
+      # CloudWatch with CloudTrail for API requests Events
+      event["eventType"]    = event.fetch_path("detail", "eventName")
+      event["event_source"] = :cloud_watch_api
+    elsif event["detail-type"] == "EC2 Instance State-change Notification"
```

Comment:
I wonder why there are so many conventions

* camelcase: `eventType` 
* underscore: `event_source` 
* dash: `detail-type`

So, it looks the `detail-type` is coming from CloudWatch, `messageType` is coming from AWS Config and `eventType` and `event_source` is something we set.

How about 
* `eventType` => `miq_aws_event_type`
* `event_source` => `miq_aws_event_source`

This makes it at least clearer, that we add the underscore keys, and we prepend them with something not already set and with `miq_aws_`